third-party/cmake/GetWindowsDeps.cmake

```diff
@@ -0,0 +1,49 @@
+# Download and install binary dependencies for windows
+include(CMakeParseArguments)
+
+# This is similar to the build recipes, but instead downloads a third party
+# binary and installs it under the the DEPS_PREFIX.
+function(GetWindowsDep)
+  cmake_parse_arguments(_gettool
+    "BUILD_IN_SOURCE"
+    "TARGET"
+    "INSTALL_COMMAND"
+    ${ARGN})
+
+  if(NOT _gettool_TARGET OR NOT _gettool_INSTALL_COMMAND)
+    message(FATAL_ERROR "Must pass INSTALL_COMMAND and TARGET")
+  endif()
+
+  string(TOUPPER "${_gettool_TARGET}_URL" URL_VNAME)
+  string(TOUPPER "${_gettool_TARGET}_SHA256" HASH_VNAME)
+  set(URL ${${URL_VNAME}})
+  set(HASH ${${HASH_VNAME}})
+  if(NOT URL OR NOT HASH )
+	  message(FATAL_ERROR "${URL_VNAME} and ${HASH_VNAME} must be set")
```

Comment:
This error would be confusing. Shouldn't it say `"${_gettool_TARGET}_URL and ${_gettool_TARGET}_SHA256 must be set"`